Porting/Maintainers.pl

```diff
@@ -488,11 +488,6 @@ package Maintainers;
             'README.packaging',
             'lib/ExtUtils/MakeMaker/version/vpp.pm',
         ],
-        'CUSTOMIZED' => [
-            # https://github.com/Perl/perl5/issues/17601
-            # https://rt.cpan.org/Ticket/Display.html?id=115321
-            't/lib/MakeMaker/Test/Setup/XS.pm',
-        ],
```

Comment:
`t/porting/customized.dat` needs to be updated too

Reply:
Got it. Thanks!